lesson1/index.html

```diff
@@ -1,17 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
IMO
It's better to change "ja".

Reply:
Thank you for your comment.
I will change it to `ja`.
57fc8a3f3636a68d1912e697cc3b6082f4e801b4